Makefile.am

```diff
@@ -1 +1 @@
-SUBDIRS = meta lib vslib syncd player
+SUBDIRS = meta lib vslib syncd player asicdump
```

Comment:
mellanox has a sdkdump to dump all sdk state, to differentiate with sdkdump, can we call it saidump? for the player, can we call it saiplayer?